eng/source-build-patches/0006-Dirty-revert-of-https-github.com-dotnet-roslyn-issue.patch

```diff
@@ -0,0 +1,25 @@
+From ee8cef65cd2bc322fe005c3c8b96ecbf39f167e0 Mon Sep 17 00:00:00 2001
+From: Chris Rummel <crummel@microsoft.com>
+Date: Fri, 4 Sep 2020 11:50:51 -0500
+Subject: [PATCH 6/6] Dirty revert of
+ https://github.com/dotnet/roslyn/issues/10785
```

Comment:
Doesn't have to be resolved right now, but @crummel what is this doing? This seems like it introduces a bug?